src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -186,6 +171,22 @@ public void updateElement(final int index, @NotNull final Pane rowPane)
         });
     }
 
+    private String createColor(final AbstractBuildingWorker.Skill primary, final AbstractBuildingWorker.Skill secondary, final AbstractBuildingWorker.Skill current)
+    {
+        if(primary == current){
+            return ColorConstants.GREEN;
+        }
+        if(secondary == current){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)